README_android.txt

```diff
@@ -7,9 +7,8 @@ This port should support Android 3.1 (Honeycomb) and above.
 Dependencies
 ============
 
-This port depends on having CMake, the Android SDK, and the Android NDK
-version r5b or better, JDK, and Apache ant. Python is required to build
-the examples.
+This port depends on having CMake, the Android SDK, the Android NDK and
+the Oracle JDK.
```

Comment:
Oracle JDK? Why?

Reply:
All of Google's Android tools run on Java including gradle, so you do need Java. In theory any JDK should work, but for me at least Open JDK just crashes when running gradle while the Oracle one works.

Reply:
Yes; I mean, I'm always using OpenJDK and staying away of Oracle's one, as it's not free, hence my question.

Reply:
I'll remove the "Oracle" - hopefully whatever crash I got is fixed in the meantime!